### PR TITLE
Used default args 

### PR DESCRIPTION
**Pull request by hritvi**
Use default args rather than "options" param in ./test/client/ #2308

### CODE REVIEW

**Comment by hritvi** [comment]
Hey @vladikoff ! I can't figure it out what went wrong.Can you please help me out in this. Thanks :)

File: test/client/index.js [hunk 1]
```diff
@@ -54,9 +54,8 @@ module.exports = config => {
     })
   }
 
-  Client.create = function (origin, email, password, options) {
+  Client.create = function (origin, email, password, options={}) {
     var c = new Client(origin)
-    c.options = options || {}
```
Comment (vladikoff) [inline test/client/index.js]: @hritvi Looks like a mistake here, you updated options, but didn't pass options to `c.options`.

File: test/client/api.js [hunk 2]
```diff
@@ -504,8 +497,7 @@ module.exports = config => {
     )
   }
 
-  ClientApi.prototype.accountUnlockResendCode = function (email, options, lang) {
-    options = options || {}
+  ClientApi.prototype.accountUnlockResendCode = function (email, options={}, lang) {
```
Comment (vladikoff) [inline test/client/api.js]: please add spaces between ` = ` in `options = {}` just like other files above

File: test/client/api.js [hunk 3]
```diff
@@ -560,8 +552,7 @@ module.exports = config => {
       )
   }
 
-  ClientApi.prototype.sessionReauth = function (sessionTokenHex, email, authPW, options) {
-    options = options || {}
+  ClientApi.prototype.sessionReauth = function (sessionTokenHex, email, authPW, options={}) {
```
Comment (vladikoff) [inline test/client/api.js]: please add spaces between ` = ` in `options = {}` just like other files above

File: test/client/api.js [hunk 4]
```diff
@@ -739,8 +730,7 @@ module.exports = config => {
     )
   }
 
-  ClientApi.prototype.verifyTokenCode = function (sessionTokenHex, uid, code, options) {
-    options = options || {}
+  ClientApi.prototype.verifyTokenCode = function (sessionTokenHex, uid, code, options={}) {
```
Comment (vladikoff) [inline test/client/api.js]: please add spaces between ` = ` in `options = {}` just like other files above

File: test/client/index.js [hunk 5]
```diff
@@ -54,9 +54,8 @@ module.exports = config => {
     })
   }
 
-  Client.create = function (origin, email, password, options) {
+  Client.create = function (origin, email, password, options={}) {
```
Comment (vladikoff) [inline test/client/index.js]: please add spaces between ` = ` in `options = {}` just like other files above

**Comment by vladikoff** [comment]
@hritvi besides test/client, please also search for the same option usage in other files and update them.

All of these and more:

image

**Pull request closed without merging**